### PR TITLE
ENH: Webpack and babel

### LINKED ISSUE #6: Support Babel transpiling from JS ES2015 to ES5

### LINKED ISSUE #20: Support webpack for building dist

### PR DESCRIPTION
**Pull request by turanoo**
Fixes #6
Fixes #20

### CODE REVIEW

**Review by thermokarst — CHANGES_REQUESTED** [review]
Hey @turan1393, it looks like this PR is missing your updated `package.json` file - can you please add that to this PR?

File: package.json [hunk 1]
```diff
@@ -5,5 +5,17 @@
   "main": "index.js",
   "repository": "https://github.com/caporaso-lab/SurveySync",
   "author": "Caporaso Lab",
-  "license": "BSD 3-Clause License"
+  "license": "BSD 3-Clause License",
```
Comment (thermokarst) [inline package.json:8]: Since we won't be publishing this on `npm`, can you add a new config item:

```json
"private": true,
```

**Review by thermokarst — CHANGES_REQUESTED** [review]
This looks great to me! One minor addition - can you update `README.md`? We have dependencies (`yarn`), and then to get the repo set up and ready to run, someone would need to run `yarn install`. If they want to `build` the code they would need to run `npm run build`. Make sense?

File: package.json (hunk 1 above)
Comment (turanoo) [reply package.json:8]: Got it!

**Review by thermokarst — APPROVED** [review]
LGTM 👍

**Pull request merged**